mock/sink.go

```diff
@@ -224,7 +248,7 @@ func (s *Sink) AssertCounterEquals(tb testing.TB, name string, exp uint64) {
 	tb.Helper()
 	u, ok := s.LoadCounter(name)
 	if !ok {
-		tb.Errorf("gostats/mock: Counter (%q): not found", name)
+		tb.Errorf("gostats/mock: Counter (%q): not found in %v", name, s.ListCounters())
```

Comment:
```suggestion
		tb.Errorf("gostats/mock: Counter (%q): not found in: %q", name, s.ListCounters())
```
Use `%q` so the keys are quoted.

Reply:
Cool, didn't know that would work on slices!